tardis/tests/test_atomic.py

```diff
@@ -38,22 +38,40 @@ def test_atomic_symbol():
 def test_atomic_symbol_reverse():
     assert atomic.symbol2atomic_number['Si'] == 14
 
-@pytest.mark.skipif(not pytest.config.getvalue("atomic-dataset"),
-                    reason='--atomic_database was not specified')
-def test_atomic_reprepare():
-    atom_data_filename = os.path.expanduser(os.path.expandvars(
-        pytest.config.getvalue('atomic-dataset')))
-    assert os.path.exists(atom_data_filename), ("{0} atomic datafiles "
-                                                         "does not seem to "
-                                                         "exist".format(
-        atom_data_filename))
-    atom_data = atomic.AtomData.from_hdf5(atom_data_filename)
-    atom_data.prepare_atom_data([14])
-    assert len(atom_data.lines) > 0
-    # Fix for new behavior of prepare_atom_data
-    # Consider doing only one prepare_atom_data and check
-    # len(atom_data.lines) == N where N is known
-    atom_data = atomic.AtomData.from_hdf5(atom_data_filename)
-    atom_data.prepare_atom_data([20])
-    assert len(atom_data.lines) > 0
+
+param_atom_num = [[14], [20], [14, 20]]
```

Comment:
Sure, but I wanted to use ids to make test names more informative.  
In the latest version of pytest ids can be either a list of string names or a function that returns names, but the version used by Travis seems only to support list of strings. So without using a global variable it could be like this: 
```
@pytest.mark.parametrize("selected_atomic_numbers", [[14], [20], [14, 20]],
  pytest.config.getvalue('atomic-dataset')))
                       ids = ["atom_num: {}".format(_) for _ in [[14], [20], [14, 20]]])
```
Without ids at all pytest doesn't show atomic numbers, only the name of the list with index:
`tardis/tests/test_atomic.py::test_prepare_atom_data_set_lines[selected_atomic_numbers0] PASSED`\
Well, that was my reasoning, but I can remove ids altogether.